style.css

```diff
@@ -0,0 +1,243 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	font-size: 62.5%;
+	background: #393634;
+}
+
+.container {
+	display: flex;
+	color: white;
+	font-size: 1rem;
+	font-family: 'Poppins', sans-serif;
+}
+
+p {
+	line-height: 1.5rem;
+	padding: 1%;
+}
+
+a {
+	text-decoration: none;
+}
+
+h1, h2, h4 {
+	color: #ff7292;
+}
+
+h3 {
+	border-bottom: dotted 0.5px #ff7292;
+	padding-bottom: 2.5%;
+	margin-left: 10%;
+	margin-right: 10%;
+}
+
+span {
+	color: white;
+}
+
+button {
+	border-radius: 100%; 
+	color: #ff7292;;
+	background: #3b3837;
+	border-color: #ff7292;
+	cursor: pointer;	
+}
+
+.menu-image {
+	width: 2.2rem;
+	height: 2.2rem;
+	margin-top: 25%;
+}
+
+.menu-logo {
+	width: 6rem;
+	height: 6rem;
+	margin-top: 30px;
+}
+
+.menu {
+	width: 15%;
+	display: flex;
+	flex-wrap: wrap;
+	border-right: solid black 1px;
+	justify-content: space-between;
+	position: fixed;
+	height: 100vh;
+	background-color: #3b3837;/*#393634*/	
+}
+
+nav {
+	display: flex;
+	flex-wrap: wrap;
+	text-align: center;
+}
+
+nav a {
+	width: 100%;
+	padding-top: 30px;
+	padding-bottom: 30px;
+	border-bottom: solid 0.5px gray;
+	margin-left: 30%;
+	margin-right: 30%;
+	color: white;
```

Comment:
text color instead of hex? its good to be consistent with color values, use all hex